utils/helpers.js

```diff
@@ -267,6 +269,19 @@ const unPauseAndPoll = async(ERC20Pausable) => {
 
     return false // paused
 }
+
+const pollForEvent = async(contract) => {
```

Comment:
It's unclear was `contract` is, please add a type definition.
In https://github.com/hashgraph/hedera-smart-contracts/pull/584/files#diff-ab063aef47051b3442202d7087747109a96c12399fd2dd7c0c92a48f2c4c32a2R271 you pass in a receipt

Reply:
Yes, renamed to the more appropriate `transactionReceipt`.  It's a JS project so I can't declare an explicit object type.